style.css

```diff
@@ -648,7 +601,105 @@ textarea::placeholder {
 
 /* side menu  */
 
+/* popup  */
+.modal-container {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(15px);
+  z-index: 100;
+  display: none;
+  justify-content: center;
+  align-items: center;
+
+  /* display: none; */
+}
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)